stdlib/public/runtime/CMakeLists.txt

```diff
@@ -18,9 +18,17 @@ if(SWIFT_RUNTIME_ENABLE_LEAK_CHECKER)
 endif()
 
 set(swift_runtime_port_sources)
-if("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN")
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
   set(swift_runtime_port_sources
+      MutexWin32.cpp
       CygwinPort.cpp)
```

Comment:
Is CygwinPort.cpp intended to be part of the MSVC build? Should it be renamed `Win32.cpp` or something more general instead?